frontend/src/metabase/dashboard/actions.js

```diff
@@ -138,12 +138,16 @@ export const FETCH_DASHBOARD_PARAMETER_FIELD_VALUES =
 export const SET_SIDEBAR = "metabase/dashboard/SET_SIDEBAR";
 export const CLOSE_SIDEBAR = "metabase/dashboard/CLOSE_SIDEBAR";
 
+export const SET_DASHBOARD_SEEN = "metabase/dashboard/SET_SEEN";
+
 export const initialize = createAction(INITIALIZE);
 export const setEditingDashboard = createAction(SET_EDITING_DASHBOARD);
 
 export const setSidebar = createAction(SET_SIDEBAR);
 export const closeSidebar = createAction(CLOSE_SIDEBAR);
 
+export const setDashboardSeen = createAction(SET_DASHBOARD_SEEN);
+
```

Comment:
I wonder if we can improve the naming around the `dashboardSeen` thing? I'm worried it won't be immediately straightforward to get the idea of it if you don't know about dynamic favicons. E.g. when I just open a dashboard and watch cards loading, will it mean that I've seen the dashboard? Maybe something like `hasSeenLoadedDashboard` would suit better? Just an idea though